### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by potomak**

### CODE REVIEW

**Review by dalpo — APPROVED** [review]

File: index.md [hunk 1]
```diff
@@ -5,4 +5,83 @@ title: issuehunter
...
+Issuehunter is a decentralized application that enables Open Source crowdfunding
+on the Ethereum blockchain.
+
+It is a new way to improve Open Source development lifecycle by making it
+transparent and deterministic.
+
+It makes it easier for supporters to funds projects they love and for Open
+Source contributors to collect rewards for their work.
+
+The dapp is composed by three main elements:
```
Comment (bugant) [inline index.md]: I'd change this to: The decentralized app (dapp)

File: index.md [hunk 2]
```diff
@@ -5,4 +5,83 @@ title: issuehunter
...
+
+* a set of smart contracts deployed on the Ethereum blockchain
+* a service that helps verifying software patches submitted
+* a web based directory to discover funded issues that are still waiting to be
+  resolved
```
Comment (bugant) [inline index.md]: Do we also have integration (an automatic comment or something like that) with github (and others to come later) to show directly on the issue page that there is an active bounty campaign on Issuehunter?

File: index.md [hunk 3]
```diff
@@ -5,4 +5,83 @@ title: issuehunter
...
+It lowers barriers to get access to funds for authors and it helps focusing the
+development of a project in specific areas for funders.
+
+## How it works
+
+### Step 1: fund an issue
+
+Contributors add funds to any unresolved issue on GitHub. The act of adding
+funds to an open issue triggers a smart contract function call and a transaction
+of Ethereum between the funder and the contract that is controlling the issue's
```
Comment (bugant) [inline index.md]: and a transaction on the Ethereum blockchain with a fund transfer between the funder and the contract...

File: index.md [hunk 4]
```diff
@@ -5,4 +5,83 @@ title: issuehunter
...
+funds.
+
+You can add funds to any issue on GitHub. If the issue was already funded by
+other contributors, your funds will add up to the total funds of the issue.
```
Comment (bugant) [inline index.md]: Otherwise, a fresh new contract will be created to collect funds to promote and sponsor the solution of the issue.

**Review by bugant — CHANGES_REQUESTED** [review]

File: index.md (hunk 2 above)
Comment (potomak) [reply index.md]: Yes, that's correct, I forgot to mention it!

File: index.md [hunk 5]
```diff
@@ -5,4 +5,83 @@ title: issuehunter
 
 # Introduction
 
-What a wonderful project!
+Issuehunter is a decentralized application that enables Open Source crowdfunding
+on the Ethereum blockchain.
```
Comment (dalpo) [inline index.md:9]: I'd put a link to the Ethereum website (https://www.ethereum.org/)

File: index.md (hunk 1 above)
Comment (potomak) [reply index.md]: Done.

File: index.md (hunk 3 above)
Comment (potomak) [reply index.md]: Done.

File: index.md (hunk 4 above)
Comment (potomak) [reply index.md]: Done.

File: index.md (hunk 5 above)
Comment (potomak) [reply index.md:9]: I'll put a link to the Ethereum website, but I wouldn't like to do it on the first sentence of the introduction to better focus the attention on the content.

Comment (dalpo) [reply index.md:9]: :+1:

**Pull request merged**